deliverables/README.md

```diff
@@ -11,17 +13,41 @@ Sign Up for Cloud9 and create a basic Rails project.
 ###Intro to Ruby
 ####Required
 
-What is an "interpreted" language?<br>
-What is IRB?<br>
-What are Objects?<br>
-What are Methods?<br>
-What are Classes?<br>
-What are Blocks?<br>
-What is an Array?<br>
-What is an Iterator?<br>
-What are hashes?<br>
-What is a library?<br>
-What is a gem?
+What is an "interpreted" language?<br>  
+a language where the code is not in the native language of the machine but is interpreted by an installed program  
+
+What is IRB?<br>  
+a read-eval-print loop/interactive programming enviroment that can be used for testing Ruby code by executing commands from individual users  
+
+What are Objects?<br>  
+objects are the building blocks of ruby code.  They're kind of like nouns, in that adjectives and verbs can easily be turned in to nouns so that other nouns can interact with them.  So everything can be an object and objects do things to objects via methods.  
+
+What are Methods?<br>  
+methods are the way that objects interact with eachother  
+
+What are Classes?<br>  
+groups of object bundled together by funcionality or similar characteristics  
+
+What are Blocks?<br>  
+The only type of code in Ruby that is not an object.  It can't be stored in a variable. They are a way of grouping  
```

Comment:
There are lots of people on the internet saying the blocks are not objects.  I wouldn't debate, because I don't understand the concepts well enough.  I think I get the "snippets of functionality that can be used repeatedly" concept though.  Thanks